src/drivers/virtio/transport/pci.rs

```diff
@@ -219,6 +220,36 @@ impl PciCap {
 	pub fn dev_id(&self) -> u16 {
 		self.origin.dev_id
 	}
+
+	/// Returns a reference to the actual structure inside the PCI devices memory space.
+	fn map_common_cfg(&self) -> Option<VolatileRef<'static, CommonCfg>> {
+		if self.bar.length < u64::from(self.length + self.offset) {
+			error!("Common config of with id {} of device {:x}, does not fit into memory specified by bar {:x}!", 
```

Comment:
The error message seems to be missing a word. "Common config of the capability with id {} of device {:x} does not fit into memory specified by bar {:x}!" maybe?

Reply:
It is not really the topic of this PR but thought we may want to fix it while we are at it.

Reply:
Done! 👍